src/app/components/world-map/world-map.component.ts

```diff
@@ -42,4 +56,20 @@ export class WorldMapComponent implements AfterViewInit {
 
     tiles.addTo(this.map);
   }
+
+  private addData(data: CovidDataPoint[]): void {
+    data.forEach(({ location, confirmed, dead, recovered }: CovidDataPoint) => {
+      const { lat, long } = byCountryCode(location);
+
+      // Draws 3 circles for each data point - recovered, dead, infected
+      // Each AREA is proportional to the number of cases
+      const recoveredRadius = SCALE_FACTOR * Math.sqrt(recovered);
+      const deadRadius = SCALE_FACTOR * Math.sqrt(dead);
+      const infectedRadius = SCALE_FACTOR * Math.sqrt(confirmed - dead - recovered);
+
+      circle([lat, long], { radius: recoveredRadius, color: RECOVERED_COLOUR }).addTo(this.map);
+      circle([lat, long], { radius: deadRadius, color: DEAD_COLOUR }).addTo(this.map);
+      circle([lat, long], { radius: infectedRadius, color: INFECTED_COLOUR }).addTo(this.map);
```

Comment:
Would it be more useful to set the colour through css and setting the [className](https://leafletjs.com/reference-1.7.1.html#circle-classname) instead?

Not really a major issue, just think it would be easier later if we want to be able to toggle the circles away. Unless leaflet has a better way.

Reply:
Good idea, done.
Figuring out to add `::ng-deep` for styling to be applied to derived classes was a pain the bum.

Reply:
cool, so i guess the `::ng-deep` similar to the `:global()` in svelte?